docs/rewards/Troubleshooting.mdx

```diff
@@ -0,0 +1,26 @@
+---
+id: troubleshooting
+title: Reward Annotations Troubleshoot
+sidebar_label: Troubleshooting
+slug: /project/troubleshooting
```

Comment:
Let's change the slug to `/project/rewards-troubleshooting` , so it can be even more clear by looking at the URL what this troubleshooting page is about.